src/styles/block/header.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  padding: 0 50px;
+  font-weight: 500;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  height: 60px;
+  margin-left: 20px;
+  align-items: center;
+  color: $primary-color;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  cursor: default;
+  justify-content: center;
+}
+
+.is-active {
+  margin-left: 0;
+  color: $blue-color;
```

Comment:
Similar to the previous comment, make sure that `$blue-color` is defined in the `variables.scss` file. If it's not defined, it will cause a compilation error.